package.json

```diff
@@ -37,7 +37,7 @@
     "url": "git://github.com/mapbox/node-sqlite3.git"
   },
   "dependencies": {
-    "nan": "~2.10.0",
+    "nan": "~2.11.1",
```

Comment:
```suggestion
    "nan": "~2.12.0",
```

Reply:
Reverted since ATM it fails on Travis with [`NODE_VERSION="6" ELECTRON_VERSION="3.0.6"`](https://travis-ci.org/mapbox/node-sqlite3/jobs/468884290)

Reply:
Sorry, I was not aware of the issue introduced in 2.12.0.